packages/contentstack-export/src/lib/export/assets.js

```diff
@@ -75,212 +79,223 @@ ExportAssets.prototype = {
                   return Promise.map(
                     assetsJSON,
                     function (assetJSON) {
-                      return self
-                        .getVersionedAssetJSON(assetJSON.uid, assetJSON._version)
-                        .then(function () {
-                          self.assetContents[assetJSON.uid] = assetJSON;
-                          // log.success(chalk.white('The following asset has been downloaded successfully: ' +
-                          //     assetJSON.uid))
-                        })
-                        .catch(function (error) {
-                          addlogs(
-                            config,
-                            chalk.red('The following asset failed to download\n' + JSON.stringify(assetJSON)),
-                          );
-                          addlogs(config, error, 'error');
-                        });
+                      if (self.assetConfig.downloadVersionAssets) {
+                        return self
+                          .getVersionedAssetJSON(assetJSON.uid, assetJSON._version)
+                          .then(function () {
+                            self.assetContents[assetJSON.uid] = assetJSON;
```

Comment:
Please make use of the latest concurrency utility

Reply:
Hi Shafeew, Asper our call this morning I'm skipping this point